### PR TITLE
Do not store time in the data_1 matrix

### PR DESCRIPTION
**Pull request by sjoelund**
Also changed the number of rows in the data_1 matrix to 1 instead of 2.
Previously, startTime/stopTtime was stored in the data_1 matrix, but
this is not really necessary.

### CODE REVIEW

**Comment by OpenModelica-Hudson** [comment]
The test suite is unstable according to [OpenModelica_TEST_PULL_REQUEST 2017-10-18_10-35-45](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/4769/).

**Comment by lochel** [comment]
Time is stored in the result files in case of partial result files, in order to see how much of the simulation has actually been accomplished.

**Comment by OpenModelica-Hudson** [comment]
The job failed to compile; for details, see [OpenModelica_TEST_PULL_REQUEST 2017-10-18_11-21-29](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/4771/console).

**Comment by OpenModelica-Hudson** [comment]
The job failed to compile; for details, see [OpenModelica_TEST_PULL_REQUEST 2017-10-18_11-32-26](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/4772/console).

**Comment by OpenModelica-Hudson** [comment]
The test suite is unstable according to [OpenModelica_TEST_PULL_REQUEST 2017-10-18_11-48-26](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/4773/).

**Comment by OpenModelica-Hudson** [comment]
The test suite is unstable according to [OpenModelica_TEST_PULL_REQUEST 2017-10-18_12-10-19](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/4774/).

**Comment by OpenModelica-Hudson** [comment]
The test suite is unstable according to [OpenModelica_TEST_PULL_REQUEST 2017-10-18_14-13-04](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/4777/).

**Comment by OpenModelica-Hudson** [comment]
The test suite is unstable according to [OpenModelica_TEST_PULL_REQUEST 2017-10-18_15-06-30](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/4779/).

**Comment by OpenModelica-Hudson** [comment]
The test suite is unstable according to [OpenModelica_TEST_PULL_REQUEST 2017-10-18_15-59-58](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/4781/).

**Comment by OpenModelica-Hudson** [comment]
The tests run correctly according to [OpenModelica_TEST_PULL_REQUEST 2017-10-18_16-56-16](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/4782/console).

File: SimulationRuntime/c/simulation/results/simulation_result_mat.cpp [hunk 1]
```diff
@@ -712,47 +725,47 @@ static void generateData_1(DATA *data, threadData_t *threadData, double* &data_1
...
-             mData->nParametersInteger +
-             mData->nParametersBoolean;
+  cols = mData->nParametersReal +
+         mData->nParametersInteger +
+         mData->nParametersBoolean;
+  if (cols) {
+    rows = 1;
+  } else {
+    rows = 0;
+  }
```
Comment (lochel) [inline SimulationRuntime/c/simulation/results/simulation_result_mat.cpp]: @sjoelund What if all of these parameters are skipped due to the variable filter? I guess we have to iterate over all the parameters twice: first to count them and second to copy them to `data_1`.

Comment (sjoelund) [reply SimulationRuntime/c/simulation/results/simulation_result_mat.cpp]: Parameters are not filtered as far as I remember

Comment (lochel) [reply SimulationRuntime/c/simulation/results/simulation_result_mat.cpp]: It looks like they get filtered:

https://github.com/OpenModelica/OMCompiler/blob/b610cb9d9d839c81faf23b32c7f67c8c06acacc2/SimulationRuntime/c/simulation/results/simulation_result_mat.cpp#L749-L755

Comment (lochel) [reply SimulationRuntime/c/simulation/results/simulation_result_mat.cpp]: Never mind. I have to change that code anyway a bit to add the time-unvarying variables to `data_1`.

**Pull request closed without merging**